fprodents/workflows/base.py

```diff
@@ -296,10 +298,25 @@ def init_single_subject_wf(subject_id):
         )
     )
 
-    for bold_file in subject_data["bold"]:
+    bold_ref_wf = init_epi_reference_wf(
+        auto_bold_nss=True,
+        omp_nthreads=config.nipype.omp_nthreads
+    )
+    bold_ref_wf.inputs.inputnode.in_files = subject_data["bold"]
```

Comment:
ha! so you want to generate a unique BOLD reference for all runs? That is cool (and definitely in the roadmap) but it feels a bit precipitated. A problem that will come up is, what if I want to run the dataset partially? first only one task and then all tasks? Results will be different from the first execution on only one task to the second.